src/utils/fetchData.js

```diff
@@ -7,14 +7,20 @@ const fetchData = async (view, benchmark, timeRange = CONFIG.default.timeRange)
   const ALL_DATA = {};
 
   const fetchIt = async (config, options) => {
-    const response = await queryPerfData(config, options);
-    if (response) {
-      Object.values(response).forEach((series) => {
-        ALL_DATA[series.meta.id] = {
-          ...series,
-          ...config,
-        };
-      });
+    try {
+      const response = await queryPerfData(config, options);
+      if (response) {
+        Object.values(response).forEach((series) => {
+          ALL_DATA[series.meta.id] = {
+            ...series,
+            ...config,
+          };
+        });
+      }
+    } catch (error) {
+      // If we don't let it propragate the rest of the app will continue to work
+      // eslint-disable-next-line no-console
+      console.error(`We failed to fetch ${config.suite}`);
```

Comment:
I suggest not swallowing errors.  I suspect something is important is missing when this happens and the user may want to know.

Usually, you can catch-and-re-raise so at least you have an explanation:

    raise new Error("failed to fetch....", {cause: error})

You may need a real error/logging library for that; I was working on a basic API for that: https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/355/files#diff-4624072ebb6d0ddde8d37b9a48842ff7R37

I seems the componentDidMount (and componetDidUpdate) is the ultimate caller of this.  Maybe the health dashboard's `withErrorBoundary` would help here.

Reply:
There's really no value to show to the user as they don't know how to debug it.
If it's reproducible we can investigate or add Sentry reporting.

For now, I'm going to land as-is as it unblocks us. If it becomes an issue we can revisit.